rbs/prism/MethodTypeToParserNodePrism.cc

```diff
@@ -621,14 +616,14 @@ pm_node_t *MethodTypeToParserNodePrism::methodSignature(const pm_node_t *methodD
 
                 // Special case: anonymous block parameter (&) should use symbol :&
                 if (arg.kind == RBSArg::Kind::Block && methodArg.nameId == PM_CONSTANT_ID_UNSET) {
-                    symbolNode = PMK::Symbol(tinyLocOffsets, "&");
+                    symbolNode = prism->Symbol(tinyLocOffsets, "&");
```

Comment:
use string view literal

```suggestion
                    symbolNode = prism->Symbol(tinyLocOffsets, "&"sv);
```

(there's a few other cases left)